test/integration/test_run_command_integration.py

```diff
@@ -463,17 +461,14 @@ def test_run_interrupted(
     _assert_exit_code(final_status, exit_code)
 
     current_statuses = 'SUBMITTED, QUEUING, RUNNING'
-    # TODO: When you call ``.cmdline`` in a ``psutil.Process`` during an iteration,
-    #       if it is a zombie it might raise a ``ZombieProcess`` error.
```

Comment:
I added a note about it, but in a test. This should fix the issue and resolve this `TODO` marker :+1: